haskell/1/index.md

```diff
@@ -183,14 +132,14 @@ executable bassbull
   default-language:    Haskell2010
 ```
 
-A few notable changes:
+A few things to note:
 
-* Set the description so Cabal would stop squawking about it.
-* Set `hs-source-dirs` to `src` so Cabal knows where my modules are.
-* Added a named executable stanza to the Cabal file so I can build a binary by that name and run it.
-* Set `main-is` to `Main.hs` in the executable stanza so the compiler knows what main function to use for that binary.
-* Set `ghc-options` to `-Wall` so we get the *rather* handy warnings GHC offers on top of the usual type checking.
-* Added the libraries our project will use to `build-depends`.
+* The description is so people could know what the package is about.
```

Comment:
This reads somewhat awkwardly. "The description tells people what the package is about" might be clearer